polyglot_piranha.pyi

```diff
@@ -164,12 +164,20 @@ class Filter:
     enclosing_node: TSQuery
     "AST patterns that some ancestor node of the primary match should comply"
     not_contains: list[TSQuery]
-    "AST patterns that should not match any subtree of node matching `enclosing_node` pattern"
-
+    "AST patterns that SHOULD NOT match any subtree of node matching `enclosing_node` pattern"
+    contains: list[TSQuery]
+    "AST patterns that SHOULD match at least one subtree of node matching `enclosing_node` pattern"
```

Comment:
In this comment lets make it clear that the filter will return true based on the values of the fields `at_least` and `at_most`